docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -120,12 +148,124 @@ To retrieve the metadata and playback information on the client side:
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_DURATION, "duration_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_DATE, "date_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_COPYRIGHT, "copyright_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_DESCRIPTION, "desc_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_TRACK_NUM, "tracknum_1");
+   mc_server_add_item_to_playlist(g_server_h, playlist_h, "1", MC_META_MEDIA_PICTURE, "picture_1");
+
+   ret = mc_server_update_playlist_done(g_server_h, playlist_h);
+   ```
+4. When no longer needed, destroy the playlist handle using the `mc_playlist_destroy()` function:
+
+   ```
+   mc_playlist_destroy(playlist_h);
+   ```
+5. Destroy the media controller server handle using the `mc_server_destroy()` function, when media controller server handle is no longer needed.
+
+   ```
+   mc_server_destroy(g_server_h);
+   ```
+
+To retrieve the playlist and metadata information on the client side:
+
+1. Create the media controller client handle using the `mc_client_create()` function:
+
+   ```
+   ret = mc_client_create(&g_client_h);
+   ```
+2. Define the callback that is invoked when the client receives the playlist.
+
+   ```
+   void
+   playlist_updated_cb(const char *server_name, mc_playlist_update_mode_e mode, const char *playlist_name, mc_playlist_h playlist, void *user_data)
+   {
+     dlog_print(DLOG_DEBUG, LOG_TAG, "Server Name: %s, Playlist Update Mode: %d, Playlist Name: %s\n", server_name, mode, playlist_name);
+   }
+   ```
+
+   If you want to use playlist handle outside, make a copy using `mc_playlist_clone()` function.
+
+3. Register the callback using the `mc_client_set_playlist_updated_cb()` function:
+
+   ```
+   ret = mc_client_set_playlist_updated_cb(g_client_h, playlist_updated_cb, NULL);
+   ```
+4. Define the callback that retreive the item from the playlist handle.
+
+   ```
+   bool
+   playlist_item_cb(const char *index, mc_metadata_h metadata, void *user_data)
+   {
+     dlog_print(DLOG_DEBUG, LOG_TAG, "Index: %s\n", index);
+   }
+   ```
+
+  If you want to use metadata handle outside, make a copy using `mc_metadata_clone()` function.
+
+5. Register the callback using the `mc_playlist_foreach_item()` function:
+
+   ```
+   ret = mc_playlist_foreach_item(playlist_h, playlist_item_cb, NULL);
+   ```
+
+6. Retreive the metadata information from the metadata handle using the `mc_metadata_get()` function:
+
+   ```
+   char *title = NULL;
+   ret = mc_metadata_get(metadata_h, MC_META_MEDIA_TITLE, &title);
+   ```
+
+7. When no longer needed, destroy the playlist handle using the `mc_playlist_destroy()` function:
+
+   ```
+   mc_playlist_destroy(playlist_h);
+   ```
+
+8. Destroy the media controller client handle using the `mc_client_destroy()` function, when media controller client handle is no longer needed.
```

Comment:
Please remove 'function' word and use colon at the end of the sentence. Suggested edit:

 Destroy the media controller client handle using `mc_client_destroy()`, when media controller client handle is no longer needed: